Formula/geph4.rb

```diff
@@ -1,8 +1,8 @@
 class Geph4 < Formula
   desc "Modular Internet censorship circumvention system to deal with national filtering"
   homepage "https://geph.io/"
-  url "https://github.com/geph-official/geph4/archive/v4.4.2.tar.gz"
-  sha256 "1294c45c642dcbc4e913549cdf12168ab5df504e83b2b9fd887e99c22609e7b7"
+  url "https://github.com/geph-official/geph4/archive/refs/tags/v4.4.3.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/geph-official/geph4/archive/v4.4.3.tar.gz"
```
We can use the shorter URL here.